src/Microsoft.TemplateEngine.Utils/PhysicalFileSystem.cs

```diff
@@ -101,5 +101,18 @@ public void SetLastWriteTimeUtc(string file, DateTime lastWriteTimeUtc)
         {
             File.SetLastWriteTimeUtc(file, lastWriteTimeUtc);
         }
+
+        public IDisposable WatchFileChanges(string filepath, FileSystemEventHandler fileChanged)
+        {
+            var watcher = new FileSystemWatcher(Path.GetDirectoryName(filepath), Path.GetFileName(filepath));
```

Comment:
minor: inconsistent use of `var`.  In some cases new code uses the Type name in others var.